.lintstagedrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "*.js": "prettier --write",
+  "*.ts": "prettier --write",
```

Comment:
I am thinking we should make a `.lintstagedrc` file in both the `/go` and `/node` directories. This would allow us to use `"*": "prettier --write"` for the node linting catching all supported file types and honoring the existing `.prettierignore`. As it stands, we won't be running prettier on json which is configured and there are probably some other filetypes as well like `md`